.travis.yml

```diff
@@ -46,11 +46,11 @@ after_success:
 
 deploy:
   provider: pypi
-  user: lnielsen
+  user: inveniosoftware
   password:
-    secure: sX/eAcV9W9vCW4fa7rL+mQ0RP9zbhM3LQzQf/A+VW56gdctBA4Ew4FxHYH0QabdZF6Bzq/TQpAQSeSqzZAu3bTBGycMhWBis5z7tIpsUthmab2UDFGKtrlnF5M0u8ZoRlccQ1RNI0FNWF8puB/siCAIZ8/Krl/cwZwFbULAbem3OHAq0Ci3yLnm0STDvjFVo1QfWI40S/JH6uSrZUfTQZnfiDoO7h4nadf9EhKqqVhC5t4QZ5FYLO+iKtfq9IaNLD9iIscDtNCJ1kWqcW33vs84swDe0P0QqDk5WtAhE/MZvfqroeCzUVvX5rkfvUuzxauc8v9GnoLfmgWvCSZZthJ3A58bMKsYFPqviUkyWj9IzRcWa/saFHnZbabaCDXhO59dF/bmxEdNnZnch7hUjWG/IZ7aYUhj1zkkbVnzLoNn0ByBJdFl0q84/ue92x3VUuiR9jy2q//HZN3INr0jsMNikmqg6JwFRH0SGWzVLFUWoLwYe0pr8eXYRh+gL/ShZZ97t8AWcIRiQoJu8K1wFaq0BK1sOQbl637//cq95s4GSauwBhSwxQcFPNhzsp+6Ac70pIYiakeY2Ri/t8hjEFAVW/Z7DTbJ5GCv8Gr8IyhHat6/H3XS58Fw5kPDJrezglOwrxcO+lJecGB7q5WKDC6zxtvw2Ze20HCH/EbaZVtM=
+    secure: V3ak8HwWUvtKL4BWGCjLAAYMd5jYijGKt7nGLw5woEjrZ/ahr5stnSF1lxuJuUuFZDHbS5rAzPO1uDZLlHig5OvUv7M/LCV+Yni1waHaooB4iqL78B8xtorlomZsMf8ffd50VghU3IO1+nAWOIunQNxjz1kAv8wLYw3IEfMz2WfzEW7CKHZohZeuX2xgmJP3iDECdCOouOCFP2hQxiyP7i+ORgrkYzqOMhiv6Z6tCXwYT4cEtw7sdnHyt67yeNsDr32WxfGZ5O1lIaopvnHHhf+AyPzvfdk2ZCjCdjXtRWG2isddxDREZL6LLMwhzQJ4G54pvnZj7Zp4WtfVYNZMKmDhJhTRBx+wG10qjxcnojLGYGfyajBLt/lNBC6t6e0aowxD6dz7R2rpMEnctjmiaYiLoM2EyCQr00aSvhCljR2ujWN217l2YENJ0hdH48VkrCGcDNedddroEcLJ4HMfI655N4XjgsAevxZQxPLS4upQ59GCt238fwGnXxT2yuH13rVIpBH7xj9apn0cgArFYjb2VMEAO513ftIIXlljfNYwEeMA0loRIXI2pbZ/vZaCJEUIfDd8NOzmT5eXblwJHH3QxgWSP9xJmT3CGhM+0r0zj+mvOrd3Ku57/CE2jo3IrBPbXlToirwYzRQt430Xfyoc91Upr9q4voYjj1OnkQw=
   distributions: "sdist bdist_wheel"
   on:
     tags: true
-    python: "2.7"
+    python: "3.6"
```

Comment:
We might need to update all new modules (rdm-records, app-rdm etc) since we have 3.5 in those. Whats the convention?